### DISCUSSION TITLE
Halloween outfits? (for monika, i mean)

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#7976] [QA - tech support + general] [answered]
when it's Halloween, do u have to be up at 12:00 am for her to unlock a new outfit? i saw someone say that. but do i have to stay up until 12 am for Monika's outfits? will it still work on Halloween if i wake up at like 7 am and open the game?

**Comment by lunulae** [comment, marked as answer]
You can open the game anytime on the day of Halloween to get a costume. You just get a small bit of extra dialogue if you had the game open at midnight, since she decides to set up the decorations then.
